README.md

```diff
@@ -2,6 +2,8 @@
 
 Office Add-ins code samples are provided in this repo to help you learn, study, and built great Office Add-ins!
 
+> Join us June 15th at 8am PT for a community panel discussion on samples. What do you want us to focus on over the next 6 months? Get the invite at https://aka.ms/officeaddinsamples-join
```

Comment:
```suggestion
> Join us June 15th at 8am PT for a community panel discussion on samples. What do you want us to focus on over the next 6 months? Get the invite at https://aka.ms/officeaddinsamples-join.
```

Reply:
Per [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=27390), should also note as AM.
```suggestion
> Join us June 15th at 8 AM PT for a community panel discussion on samples. What do you want us to focus on over the next 6 months? Get the invite at https://aka.ms/officeaddinsamples-join.
```